README.md

```diff
@@ -42,18 +42,42 @@ Running `python square.py --num 2` will print `The square of your number is 4.0.
 Tap requires Python 3.8+
 
 To install Tap from PyPI run: 
+
 ```
 pip install typed-argument-parser
 ```
 
-To install Tap from source, run the following commands:
+<details>
+<summary>To install Tap from source, run the following commands:</summary>
 
 ```
 git clone https://github.com/swansonk14/typed-argument-parser.git
 cd typed-argument-parser
 pip install -e .
 ```
 
+</details>
+
+<details>
+<summary>To develop this package, install development requirements (in a virtual environment):</summary>
+
+```
+python -m pip install -e ".[dev]"
+```
+
+Style:
+- Please use [`black`](https://github.com/psf/black) formatting
```

Comment:
Pretty sure this is [accurate](https://github.com/swansonk14/typed-argument-parser/commit/52df386df01a4bb9e060d366509fed470da55160)